gammapy/modeling/models/spatial.py

```diff
@@ -1554,6 +1563,15 @@ def evaluate(self, lon, lat, energy=None, **kwargs):
         return u.Quantity(val, self.map.unit, copy=False)
 
     def write(self, overwrite=False):
+        """
+        Write the map.
+
+        Parameters
+        ----------
+        overwrite: bool, optional
+            Overwrite existing file.
+            Default is False, which will raise a warning if the template file exists already.
```

Comment:
Maybe this can be explained in the main part of the docstring or as a note since this is a different behavior compared to most cases: i.e. it does not fail if there is an existing file and `overwrite=False`.